xDripG5/Messages/SessionStartTxMessage.swift

```diff
@@ -12,11 +12,16 @@ import Foundation
 struct SessionStartTxMessage: RespondableMessage {
     typealias Response = SessionStartRxMessage
 
-    let startTime: UInt32
+    /// Time since activation in Dex seconds
+    let time: UInt32
+
+    /// Time in seconds since Unix Epoch
+    let timeEpoch: UInt32
```

Comment:
The platform epoch (known as the [reference date](https://developer.apple.com/documentation/foundation/date)) is actually January 1, 2001. A better name for this would be `secondsSince1970`.

Reply:
Done.